src/TransitionReplace/index.jsx

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
+import classNames from 'classnames';
+
+
+const propTypes = {
+  children: PropTypes.element,
```

Comment:
Should this be ```required``` given that without children this component wouldn't really do anything?

Reply:
We have a current use-case in the profile app where we don't always supply children. It works fine without it – effectively it becomes a transition in from 0 height.

Reply:
Interesting, it may be worth adding an example to storybook.

Reply:
Actually just double checked this. The reason for allowing no children was to make it easier to work with async content. It won't do a transition in height from nothing. If we want it to do that we'll need to add some more to this component in the future.

Reply:
Cool, it would be great to update the readme to note this for clarity.

Reply:
All set